src/sentry/static/sentry/app/views/releases/releaseCommits.jsx

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
+
 import LoadingIndicator from '../../components/loadingIndicator';
 import LoadingError from '../../components/loadingError';
+import Avatar from '../../components/avatar';
+import TimeSince from '../../components/timeSince';
+
 import ApiMixin from '../../mixins/apiMixin';
 
 const ReleaseCommit = React.createClass({
   propTypes: {
     commitId: React.PropTypes.string,
     shortId: React.PropTypes.string,
     commitMessage: React.PropTypes.string,
-    commitDateCreated: React.PropTypes.string
+    commitDateCreated: React.PropTypes.string,
+    author: React.PropTypes.object,
+    repository: React.PropTypes.object,
+
   },
   render() {
     return (
       <li className="list-group-item" key={this.props.commitId}>
-        <div className="row">
-          <div className="col-sm-2 col-xs-2"><strong>{this.props.shortId}</strong></div>
-          <div className="col-sm-7 col-xs-7">{this.props.commitMessage}</div>
-          <div className="col-sm-3 col-xs-3 align-right actions">{this.props.commitDateCreated}</div>
+        <div className="row row-center-vertically">
+          <div className="col-xs-8 list-group-avatar">
+            <Avatar user={this.props.author}/>
+            <h5>{this.props.commitMessage}</h5>
+            <p><strong>{this.props.author.name}</strong> committed <TimeSince date={this.props.commitDateCreated} /></p>
+          </div>
+          <div className="col-xs-2"><span className="repo-label">{this.props.repository.name}</span></div>
+          <div className="col-xs-2 align-right">
+            <a className="btn btn-default btn-sm"
+               href={this.props.repository.url + '/' + this.props.commitId}
+               target="_blank"><span
+               className={'icon-mark-' + this.props.repository.provider.id}/>&nbsp; {this.props.shortId}</a>
```

Comment:
Nitpick: I probably would have aliased `props` rather than access `this.props` 10 times.